### PR TITLE
Bug 1237223 - [TV] FxOS TV Metrics - The usage of the preview window from Marketplace is not recorded by metrics

### PR DESCRIPTION
**Pull request by luke-chang**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=295a99059789b7d40d809583e70468a07fb71767

File: tv_apps/smart-system/js/preview_window.js [hunk 1]
```diff
@@ -25,7 +25,7 @@
       this.container = configs.rearWindow;
     }
```
Comment (rexboy7) [inline tv_apps/smart-system/js/preview_window.js]: Although it's not related with this patch..
`This window is inherit the AppWindow` -> `This window inherits AppWindow`

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d650d18ebc99b1f1e7c222b303c370268f0c4c2e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=9f68fa822c1f550281647e13efddbdcb84235e35

**Pull request merged**